src/app/reducers/vm/redux/vm.reducers.ts

```diff
@@ -287,7 +287,7 @@ export const selectVmGroups = createSelector(
 export const getUsingSGVMs = createSelector(
   selectAll,
   fromSGroup.getSelectedId,
-  (vms, sGroupId) => {
+  (vms: VirtualMachine[], sGroupId: string) => {
```

Comment:
I was talking about arrow function. You did not have any troubles with this params